packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -0,0 +1,241 @@
+import { useRouter } from "next/router";
```

Comment:
Isn't this and the `profile.tsx` file mainly copied from teams? Can't we make a reusable component here? I might be missing something

Reply:
I will do this in a follow up - as there is likely a few settings pages that can take advantage of this

Reply:
Agreed with Carina. Let's try to get this as DRY as possible.

Reply:
Agreed - in another PR 

> Agreed with Carina. Let's try to get this as DRY as possible.